docs/advanced/experimental/indexers-cheat-sheet.md

```diff
@@ -0,0 +1,43 @@
+# Indexers on Jina Hub
+
+The [Jina Hub](hub.jina.ai) offers multiple Indexers for different use-cases.
+There are four major questions that should be answered, when deciding for an Indexer and its configuration.
+In a lot of production use-cases Indexers heavily use {ref}`shards <./scaling/shards>` and {ref}`replicas <./scaling/replicas>`
+
+## Does my data fit into memory?
+
+What is your estimated total number `N` of Documents, that you want to index?
+What is the average size `x` of a Document?
+Does `N * x` fit into memory?
+
+## How many requests per second (RPS) does the system need to handle?
+
+RPS is typically used for knowing how big to scale distributed systems.
+Depending on your use-case you might have completely different RPS expectations.
```

Comment:
rps actually depends on how many requests the indexers can handle in parallel. I don't think it is directly related to the indexer.

Reply:
Jina Executors can never handle requests in parallel. So when you need a certain throughput, you need to consider your indexer configuration.